spec/support/database_cleaner.rb

```diff
@@ -0,0 +1,18 @@
+require 'database_cleaner'
```

Comment:
@jtapia is this file required? It's not clear to me the reason why it was added, as the commit message only states `Add database_cleaner configuration`. 

Can you change the commit message to include an explanation? That would be great for documentation purposes.

Reply:
I added this configuration file because the gem is required but never used, here https://github.com/solidusio/solidus_auth_devise/blob/master/solidus_auth_devise.gemspec#L31

Reply:
ok, I just removed the commit that included it